src/main/kotlin/com/sovdee/skriptbounds/bounds/AxisAlignedBox.kt

```diff
@@ -81,13 +98,19 @@ data class AxisAlignedBox(val min: Vector3d, val max: Vector3d) : BoundingBox {
     override fun expandUniform(amount: Double) {
         min.add(-amount, -amount, -amount)
         max.add(amount, amount, amount)
```

Comment:
You might be interested in using an operator function (as a receiver function on Vector3d) to have this syntax:
```suggestion
        max += amount
```
That one specifically would be:
```kt
operator fun Vector3d.plusAssign(amount: Int) {
    add(amount, amount, amount)
}
```